kubernetes/kubernetes_test.go

```diff
@@ -379,6 +410,77 @@ func TestCollectMetrics(t *testing.T) {
 			}
 		}
 	})
+	t.Run("Ensure collection occurs with parseMetrics is disabled"+
+		" ensure sensitive data is not stripped", func(t *testing.T) {
+
+		filepath.Walk(ka.msExportDirectory.Name(), func(path string, info os.FileInfo, err error) error {
```

Comment:
Where is this sourcing data from? The output of a previous test?

Reply:
The source data comes from the mock informers that are created in the bottom of the test file.
https://github.com/cloudability/metrics-agent/blob/8916f8c744d2a3e96372aa7494e8ce2b8d483537/kubernetes/kubernetes_test.go#L574

Reply:
https://github.com/cloudability/metrics-agent/blob/3ab1bd801505e77b7fa2cc2f6c08b57afaa49d7a/testdata/pods.jsonl#L2-L1